packages/core/src/schema.ts

```diff
@@ -33,6 +33,7 @@ const _schema = [
                 default: () => +new Date(),
             },
             ['postId', 'String'],
+            ['commentId', 'String', { optional: true }],
```

Comment:
ah this is confusing. A post has `_id` and `postId` and a comment has a `_id` and a `commentId`. Instead can we call this `onchainId` or something like that for both?